packages/core/src/components/helpcenter-panel/helpcenter-panel.tsx

```diff
@@ -54,19 +54,22 @@ export class HelpcenterPanel {
           type="button"
           onClick={this.openClick}
           class={`open-button ${this.isOpen}`}
+          aria-expanded="false"
+          aria-haspopup="dialog"
         >
           <dso-icon icon="help"></dso-icon>
           <span>{this.label}</span>
         </button>
         <div class={`wrapper ${this.visibility}`}>
-          <div class="dimscreen" />
-          <div class={`iframe-container ${this.slideState}`}>
+          <div class="dimscreen" onClick={this.closeClick} />
+          <div class={`iframe-container ${this.slideState}`} aria-live="polite">
```

Comment:
Werkt `aria-live="polite"` als de content die wijzigt IN het `<iframe>` zit? Ik lees op het internet ook dat inhoud dan per ongeluk 2x wordt voorgelezen (maar misschien is dat oud nieuws hoor, zie https://github.com/FreedomScientific/VFO-standards-support/issues/111)

Reply:
Voorstel om dit voorlopig zo implementeren en op een later moment zo nodig corrigeren. @MichielDHVD is het ermee eens.